### ISSUE TITLE
scss compiling error on building

### ISSUE THREAD

**Issue by woniesong92** [meteoric/meteor-ionic#268]
```
=> Errors prevented startup:                  
   
   While building the application:

   /Users/name/seriousProjects/app-ionic/client/stylesheets/app.scss:6:9:
   Scss compiler error: file to import not found or unreadable:
   ../../.meteor/local/build/programs/server/assets/packages/meteoric_ionic-sass/_ionic
```

Does anyone else get this error?

**Comment by jinsongmu** [comment]
what's in your app.scss? 
make sure it look something like this

@import '.meteor/local/build/programs/server/assets/packages/meteoric_ionic-sass/ionic';
@import 'http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css';

**Comment by woniesong92** [comment]
It has this instead:

@import '../../.meteor/local/build/programs/server/assets/packages/meteoric_ionic-sass/_ionic';
@import '../../.meteor/local/build/programs/server/assets/packages/meteoric_ionicons-sass/_ionicons';

**Comment by jorjordandan** [comment]
I had this twice,

**Comment by nickw** [comment]
This is a known issue with Meteor that will be solved in an upcoming update.

**Comment by rohanray** [comment]
I still see this issue even after upgrading to meteor 1.1.0.3:

	git@erecaho-gitlab:~/erecaho-build/gitlab-vd-master-source$ meteor build .
	Errors prevented bundling:
	While building the application:

	/home/git/erecaho-build/gitlab-vd-master-source/client/stylesheets/app.scss:2:9:
	Scss compiler error: file to import not found or unreadable:
	.meteor/local/build/programs/server/assets/packages/meteoric_ionic-sass/ionic
	Current dir:
	/home/git/erecaho-build/gitlab-vd-master-source/client/stylesheets/


	git@erecaho-gitlab:~/erecaho-build/gitlab-vd-master-source$ meteor update
	gitlab-vd-master-source: updated to Meteor 1.1.0.3.

	Changes to your project's package version selections from updating package
	versions:

	aldeed:collection2       upgraded from 2.3.3 to 2.5.0
	less                     removed from your project
	matb33:collection-hooks  upgraded from 0.7.13 to 0.7.14
	meteoric:ionic           upgraded from 0.1.17 to 0.1.19
	useraccounts:core        upgraded from 1.11.1 to 1.12.3
	useraccounts:ionic       upgraded from 1.11.1 to 1.12.3

	git@erecaho-gitlab:~/erecaho-build/gitlab-vd-master-source$ meteor build .
	Errors prevented bundling:
	While building the application:

	/home/git/erecaho-build/gitlab-vd-master-source/client/stylesheets/app.scss:2:9:
	Scss compiler error: file to import not found or unreadable:
	.meteor/local/build/programs/server/assets/packages/meteoric_ionic-sass/ionic
	Current dir:
	/home/git/erecaho-build/gitlab-vd-master-source/client/stylesheets/

**Comment by rjakobsson** [comment]
@rohanray I'm guessing that @nickw is referring to the upcoming Meteor 1.2 update, which should allow for custom operations at build, see https://quip.com/RXFlAk9Rc2xI .

**Comment by rohanray** [comment]
@rjakobsson Thanks for the info! As of now, I have moved all meteoric related assets viz. meteoric_ionic-sass & meteoric_ionicons-sass inside my own project. In <meteor_proj_folder>/client/styles/app. scss I have changed the import as below:

    @import '../ionic/meteoric_ionic-sass/ionic';
    @import '../ionic/meteoric_ionicons-sass/ionicons';

This is a workaround while I await for the upgrade. Not sure, if the meteoric package too needs an upgrade to run with Meteor 1.2.
